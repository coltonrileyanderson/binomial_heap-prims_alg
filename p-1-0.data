4 3 432 ; 0 8 102 ; 0 8 36 ; 6 8 338 ; 0 9 106 ; 

