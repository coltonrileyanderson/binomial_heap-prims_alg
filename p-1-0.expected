0: 4
1: 3(4)432
weight: 432
